app/src/test/java/org/simple/clinic/registration/phone/RegistrationPhoneScreenControllerTest.kt

```diff
@@ -41,9 +45,9 @@ class RegistrationPhoneScreenControllerTest {
   }
 
   @Test
-  fun `when screen is created and a local user is present, then the local user must be cleared`() {
+  fun `when screen is created and existing ongoing entry is present, then the local user must be cleared`() {
```

Comment:
@vinaysshenoy Can you please review this test? It seems like the test was... wrong? But it still passed?